src/index.js

```diff
@@ -1 +1,58 @@
 import './style.css';
+
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+const gameID = 'geNkiCY06ZXBLWm1WIb4';
+const requestedURL = `${baseURL}/games/${gameID}/scores`;
+const btnRefresh = document.querySelector('.btn-refresh');
+const scoreList = document.querySelector('.score-list');
+const btnSubmit = document.querySelector('.btn-submit');
+
+const renderScores = (items) => {
+  let listItem = '';
+  items.forEach((item) => {
+    listItem += `<li>${item.user}: ${item.score}</li>`;
+  });
+  scoreList.innerHTML = listItem;
+};
+
+const getScores = async () => {
+  await fetch(requestedURL)
+    .then((response) => response.json())
+    .then((json) => {
+      renderScores(json.result);
+    });
+};
+
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻  However According to the project requirement,  you should Use `async` and `await` with the `fetch` method to send and receive data. You're on the good way but you only need to improve you syntax
- Syntax: Follow this syntax which we could use in order to execute an asynchronous method async-await:

```
const getScores = async () => {
  try{
    let response = await fetch(requestedURL);
        let data = await response.json();
        renderScores(data.result);
  } catch(error){
    console.log(error.message)
  }
};

```
- Replace your code with this one 🙃 below, I've already tested 👍🏻 
- For more documentation you can check  [Promises vs Async Await](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/)

Reply:
@BenMukebo 
I followed this documentation:
https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/articles/how_to_consume_api.md
which is mentioned here:
https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/lessons/working_with_apis.md

They did not mention using try-catch. I will do the changes as you suggested but I am a little confused. I checked with my other coding partners and all followed the same instructions as mentioned.

If I used console.log then I got linters error -> "using console"
If I don't use then it again gives me linters error -> "empty block"

What should I do?

I appreciate your help and feedback.

Reply:
Hello @gak92,  You don't necessarily need to use try and catch this was just an example and if you don't need to use try and catch your code might look like:


```
const getScores = async () => {
    let response = await fetch(requestedURL);
        let data = await response.json();
        renderScores(data.result);
  } 
```

Reply:
@BenMukebo 
Thanks
I updated my code as suggested.